src/style.css

```diff
@@ -1 +1,67 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.page {
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
```

Comment:
You have used a top margin on the .search-bar element. Be consistent with your margins. Try to use only top or bottom margins, but not both, to avoid potential margin collapse issues.